cvmfs/magic_xattr.cc

```diff
@@ -480,6 +482,31 @@ std::string ProxyMagicXattr::GetValue() {
   }
 }
 
+static std::string ListProxy(download::DownloadManager *dm) {
+  vector< vector<download::DownloadManager::ProxyInfo> > proxy_chain;
+  unsigned current_group;
+  dm->GetProxyInfo(
+    &proxy_chain, &current_group, NULL);
+  std::string buf = "";
+  for (std::vector<vector<download::DownloadManager::ProxyInfo>>::iterator
```

Comment:
That's fine but wouldn't it be easier to use an integer and iterate from zero to `vector::size()`?